test/fixtures/users.yml

```
@@ -449,6 +449,30 @@ jobseeker: #就活希望するユーザー
   last_activity_at: "2020-01-01 00:00:12"
   sent_student_followup_message: true
 
+jobseeking: #就職活動中の卒業生
```

```suggestion
jobseeking: # 就職活動中の卒業生
```

慣習でコメントの後は1スペース開けるのがいいかなと思います。